com.endlessm.HackSoundServer.json.in

```diff
@@ -0,0 +1,27 @@
+{
+    "app-id": "com.endlessm.HackSoundServer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=session-bus",
```

Comment:
Does it need full access to the session bus? If possible, this should grant more targeted access to what names it may own and what names it may talk to.